lib/collection.js

```diff
@@ -804,7 +804,7 @@ Collection.prototype.updateMany = function(filter, update, options, callback) {
  * Updates documents.
  * @method
  * @param {object} selector The selector for the update operation.
- * @param {object} document The update document.
+ * @param {object} update The update operations to be applied to the documents
```

Comment:
I'm not sure this does a better job of describing what's going on, how about:
```
* @param {object} updateDocument A document containing the update operations to be applied to the selected documents.
```

Reply:
Thanks for changing that! Unfortunately, our documentation could use some love 😄